src/main/java/uk/gov/mca/beacons/service/beacons/BeaconsRelationshipMapper.java

```diff
@@ -96,33 +92,30 @@ public Beacon getMappedBeacon(
     final var owners = personsGroupedByType.get(PersonType.OWNER);
     if (owners == null) return emptyMap();
 
-    final var ownersGroupedByBeaconId = owners
+    return owners
       .stream()
+      .filter(person -> Objects.nonNull(person.getBeaconId()))
```

Comment:
Necessary to filter out `person`s that have no `beacon_id` as we're partway through refactoring from:

- A `person` of type `PersonType.OWNER` has a `beacon_id`

To:

- A `beacon` has an `owner` foreign key to `person.id`